### PR TITLE
Add comprehensive test suite for type printer module

### PR DESCRIPTION
**Pull request by sergey-shandar**
## Summary
This PR adds a complete test suite for the type printer module, covering all public functions and their various configurations.

## Key Changes
- Added `test.f.ts` with comprehensive test coverage for:
  - **Primitive types**: Tests for null, bigint, string, number (finite and infinite), undefined, and boolean values
  - **Union types**: Tests for empty, single, and multiple type unions
  - **Printer functionality**: Tests for both readonly and mutable variants of:
    - Tuple formatting
    - Struct/object formatting
    - Array formatting
    - Record formatting

## Notable Implementation Details
- Tests are organized hierarchically by feature (primitive, union, printer)
- Readonly and mutable variants are tested separately to ensure correct modifier output
- Each test validates the exact string output format expected from the printer functions
- Tests cover edge cases like infinite numbers and empty unions

https://claude.ai/code/session_01AGSnvDTT2rPcXU5baMBWdT

### CODE REVIEW

File: fs/types/ts/test.f.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import { printer, primitive, union } from './module.f.ts'
+
+const ro = printer()
+const mut = printer(true)
+
+export default {
+    primitive: {
```
Comment (sergey-shandar) [inline fs/types/ts/test.f.ts]: Don't use `export default` for new tests, use `export const` instead.

File: fs/dev/tf/module.ts [hunk 2]
```diff
@@ -108,7 +108,8 @@ export const run = async(): Promise<void> => {
     const moduleMap = await fromIo(io)(loadModuleMap(io.process.env))
     for (const [i, v] of Object.entries(moduleMap)) {
         if (isTest(i)) {
-            framework(i, scanModule(['', v.default, false]))
+            const value = 'default' in v ? v.default : v
+            framework(i, scanModule(['', value, false]))
```
Comment (sergey-shandar) [inline fs/dev/tf/module.ts]:
```suggestion
            framework(i, scanModule(['', v, false]))
```
